testing/analyzertest.cc

```diff
@@ -0,0 +1,208 @@
+#include "common.hh"
+#include "printer.hh"
+
+#include "game/analyzer.hh"
+
+struct UnitTest_Analyzer : public testing::Test {
+	float const pi2 = static_cast<float>(M_PI * 2.0);
```

Comment:
`M_PI` is undeclared and gives an error on windows: `\performous\performous\testing\analyzertest.cc(7): error C2065: 'M_PI': undeclared identifier`

According to the docs Math constants aren't defined: https://docs.microsoft.com/en-us/cpp/c-runtime-library/math-constants?redirectedfrom=MSDN&view=msvc-170

gcc however does define them so no error on linux/osx side.

c++20 comes with the numerics header in which pi is defined: https://en.cppreference.com/w/cpp/numeric/constants

Reply:
I added `constexpr` to define pi and pi2. Another solution could be a define of _USE_MATH_DEFINES: https://stackoverflow.com/questions/1727881/how-to-use-the-pi-constant-in-c